src/components/views/BlogArticle/BlogArticle.js

```diff
@@ -1,9 +1,45 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './BlogArticle.module.scss';
+import LatestBlog from '../../features/LatestBlog/LatestBlog';
 
-const BlogArticle = () => <div className={styles.root}>This is BlogArticle</div>;
+const BlogArticle = ({ articles }) => {
```

Comment:
Nie możesz korzystać z oprsa `articles` jeżeli go w żaden sposób nie dostarczasz do komponenetu.

Ale przede wszystkim to zadanie dotyczy pojedyńćzego artykułu bloga anie listy artykułów. 

W App.js mamy definicję routingu do tego widoku `<Route exact path={'/blog/:blogId'} component={BlogArticle} />`

Na tej podstawie musisz znaleźć artykuł o podanym w parametrze `id` i przekazać go do wyświetlania w komponenecie BlogArticle

Reply:
zaraz do tego siadam :)